extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js

```diff
@@ -121,17 +131,19 @@ export class AmpLightboxViewer extends AMP.BaseElement {
    * @param {!string} label Text of the button for a11y
    * @param {!string} className Css classname
    * @param {!function()} action function to call when tapped
+   * @param {opt_parent=} opt_parent parent element that append the button to
```

Comment:
let's not add any buttons to the gallery container and keep all the buttons at same level and use css to hide/show what makes sense.